packages/dx-react-scheduler-material-ui/src/templates/appointment-tooltip/command-button.jsx

```diff
@@ -25,20 +30,22 @@ const getIcon = (id) => {
 };
 
 export const CommandButtonBase = ({
-  classes,
-  className,
-  id,
-  onExecute,
+  classes, className, id, onExecute,
   ...restProps
-}) => (
-  <IconButton
-    className={classNames(classes.button, className)}
-    onClick={onExecute}
-    {...restProps}
-  >
-    {getIcon(id)}
-  </IconButton>
-);
+}) => {
+  const isOpenCommandButton = id === OPEN_COMMAND_BUTTON;
+  const buttonClass = classes[!isOpenCommandButton ? 'button' : 'floatButton'];
+  const commonProps = {
+    onClick: onExecute,
+    className: classNames(buttonClass, className),
+    ...restProps,
+  };
+  return (
+    !isOpenCommandButton
+      ? <IconButton {...commonProps}>{getIcon(id)}</IconButton>
+      : <Fab color="primary" size="small" {...commonProps}>{getIcon(id)}</Fab>
```

Comment:
Can we invert the condition to avoid the logical negation `!isOpenCommandButton`?